[READY]  Revamps MetaStation Cargo

Son-of-Space opened the pull request:
**God Save the Quartermaster**

## About The Pull Request

This PR overhauls cargo, giving most spaces more space to operate (Including Maintenance), better functionality, more realistic positioning, and interesting new layouts without changing overall footprint. I will detail each room change below:

Overall Change:
![image](https://user-images.githubusercontent.com/63861499/79624632-ecb4ce00-80f0-11ea-89a4-dbd119f96a7c.png)
As you can see, there is quite a lot of change.

Delivery Room:
![image](https://user-images.githubusercontent.com/63861499/79624640-0bb36000-80f1-11ea-9b0f-f2475907b24c.png)
Now Cargo Techs can deliver in style in many different ways shapes and forms! The mail room now looks more like an old pneumatic mailroom with prominent public-facing positions and a large central workplace. Notable additions include express delivery chutes to each department (Example below), a better, prominent crate return system, and easier-to-manage trash sorting conveyor.

Distribution:
![image](https://user-images.githubusercontent.com/63861499/79624673-53d28280-80f1-11ea-88c7-8d44659063e2.png)
These express delivery chutes have been rigorously tested by our engineers and have been given an A+ Nanotrasen Safety Rating!

Quartermaster Office:
![image](https://user-images.githubusercontent.com/63861499/79624686-76fd3200-80f1-11ea-9e31-8148b053a2d4.png)
The Quartermaster, having access to all of the resources of the station, may have splurged a little on their room. They now have more direct view on the prominent areas of their department and a much homier, open feel to their office. (Don't worry about the rocket, it's a dud... I think!)

Mining Office:
![image](https://user-images.githubusercontent.com/63861499/79624734-baf03700-80f1-11ea-8e1a-bf0d149dd351.png)
The Mining Office has been given a little bit more open room and bigger doors to bring in heavy objects. The displays have a direct viewport over the mining shuttle and their own little nook.

Security and the Warehouse:
![image](https://user-images.githubusercontent.com/63861499/79624757-eb37d580-80f1-11ea-8e44-2c426d495082.png)
Security, as representatives tasked with overseeing any potentially illegal action in cargo, now have more space to view the goings-on of the department instead of the small, out of the way space they had before. Now it really feels like an outpost.

The Warehouse has been promoted from a back-room shady spot to a place which has a lot of potential as a real staging area. Of course there will need to be a lot of renovation first, but now cargo can put some of the resources available to them on display. If you want to use the warehouse for... less scrupulous activities, there's still a space outside of view for those of you who need discretion.

Primary Storage and Transport:
![image](https://user-images.githubusercontent.com/63861499/79624804-4964b880-80f2-11ea-9652-a5b84489ef1c.png)
Tool storage has become a much more open (Yet tighter) space! The tools you know and love are still prominent, but now assistants may have to work a bit harder to get a full set of gear. The YouTool is now displayed proudly on the outer portion of tool storage where it is easily accessible.

The MULEbots have been placed on the end of the department's normal flow. NT statistics show that crews have misused MULEbots in the past, but they are still available with their own convenient staging room!

Cargo Bay:
![image](https://user-images.githubusercontent.com/63861499/79624865-c98b1e00-80f2-11ea-8470-ab06f30c81b0.png)

Cargo Bay maintains its open, standardized layout without sacrificing any functionality. Besides a decal face-lift and a few extra pieces to give the room a better atmosphere, it has gone unchanged... just the way you like it. The ORM is in a convenient spot for deposit, pickup, or re-location if you're stingy or paranoid!

Maintenance has also been slightly expanded and changed to allow for a teensy bit more view of the AI Upload and a bigger footprint for chases.


## Why It's Good For The Game

The layout presented has a lot of roleplay potential, dynamic gameplay options, and a better functional layout for players to use. There have been some small additions to add more character to the department and gives players a larger footprint to use without making the department seem too large. There are many quality of life improvements with this change, especially with regards to the express delivery chutes, crate returns, mail room location, and warehouse prominence which should naturally increase usage of some of the less-used mechanics in the department. Available rooms should be much more widely used and less ignored.

## Changelog
:cl:
add: Completely New Cargo Layout
del: The Small Security Room by the Vault has been replaced with Maintenance
tweak: Crate Returns are now easier to find and use
tweak: Tool Storage is smaller but more interesting!
tweak: The Mail Room and Offices have been combined
balance: Cargo Warehouse is now more prominent and easier to break into
balance: MULEs now have their own isolated room.
balance: There are now easier ways to mail "packages" to departments
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Son-of-Space committed:
Revamps MetaStation Cargo

God Save the Quartermaster

Son-of-Space committed:
Chair Fix

One of the chairs was Iron and not the regular chair.

MrDoomBringer commented:
Based PR. Maybe add some north-west windows to the cargo sec office so they can see into the mining lobby? 
![image](https://user-images.githubusercontent.com/29008542/79627037-0c54f200-8103-11ea-89f9-c72ac8ab1057.png)

bluetshirtguy commented:
I might just be me but it looks like there is a lack of security in the delivery room? and the orm is an easy access to the entire cargo bay.

ArcaneMusic commented:
One quick thing, there's shipping boxes in both the tool storage and cargo on most maps, which are the only 2 mapped instances of sales taggers on meta. Make sure to throw one somewhere inside.

Tlaltecuhtli commented:
why normal windows ?

Cartographer-D commented:
> 
> 
> One quick thing, there's shipping boxes in both the tool storage and cargo on most maps, which are the only 2 mapped instances of sales taggers on meta. Make sure to throw one somewhere inside.

I made sure to give shipping boxes to both places, no worries!

Cartographer-D commented:
> 
> 
> why normal windows ?

I believe normal windows had their values changed to be balanced for placement anywhere but high-security areas, thus normal windows.

trollbreeder commented:
> > why normal windows ?
> 
> I believe normal windows had their values changed to be balanced for placement anywhere but high-security areas, thus normal windows.

Reinforced windows are standard in a map, as normal ones break really easily (and should only be used for public areas). 5-6 taps with a toolbox and that window is gone.

FloranOtten commented:
That cargo delivery system will be used more for maxcapping than for cargo. It ends in a public area, so any special items will be stolen instantly, and if you need to wait at the exit, you might as well just go get it from cargo yourself. Now, imagine someone being able to take their bombs and risk-freely bomb security. Then engineering. Then Service. Then Science. Then Medbay.

Cartographer-D commented:
> 
> 
> That cargo delivery system will be used more for maxcapping than for cargo. It ends in a public area, so any special items will be stolen instantly, and if you need to wait at the exit, you might as well just go get it from cargo yourself. Now, imagine someone being able to take their bombs and risk-freely bomb security. Then engineering. Then Service. Then Science. Then Medbay.

You can easily maxcap a department using a package wrap and destination tagger anyways. This can easily happen.

For the special items, the point is to actually tell someone that a package was sent (And/Or go with the package)

EOBGames commented:
I'm not sure if there's something I'm missing in the screenshot, but the crate return system looks like a massive security flaw- all you'd need to do is crawl under the flaps to get in.

I'm not sure I like the placement of the mulebots (they already get little to no use when they're easily available, relegating them to a corner seems like it would put them firmly in the "never use" category for most players. Also something about the tool storage is throwing me, I think it's the vendors being oddly placed. I feel like that area could use some tweaking.

Other than that it's an interesting redesign and you've certainly got a good eye for detail.

Cartographer-D commented:
I will change the outside windows to reinforced until someone fixes the map as a whole.

I will add a windoor to the crate returner, woe be to those who crawl down and get stuck.

I will check the idea of adding another window to the security desk but I don't want to go window overkill so it likely won't be added.

Son-of-Space committed:
Suggested Changes

Changed Outside windows to Reinforced. Added a windoor to crate returns. Populated maintenance next to tool storage.

TheVekter commented:
This is your first PR? You started out with a major project and ended up nailing it hardcore.

Can't _wait_ for this to get merged. So cool.

Rohesie commented:
Very nice. Let us know when you feel this is ready.

Yenwodyah commented:
The shady backroom warehouse was good 👎

trollbreeder commented:
I'm very not sure about the mulebot staging room and tool storage.
I suggest you remove the staging room, move the mulebots into cargo (replacing the 2 tables near the crate recycling belt), and expand the tool storage upwards.

Here's a general idea of what i think tool storage should look like. You can get rid of the middle table and make the room less wide while readding the maintenance path if you want.
![image](https://user-images.githubusercontent.com/25937686/79694746-9374a800-8272-11ea-91c7-51cb717b03fe.png)

Cartographer-D commented:
> 
> 
> Very nice. Let us know when you feel this is ready.

I am all set to merge when you guys are. If someone feels like they want to run it over in a map editor and double-check the cables, atmos, and pipes I would appreciate it but it likely isn't needed.

Cartographer-D commented:
> 
> 
> The shady backroom warehouse was good 👎

It'll still be pretty easy to seal it up and make it a pretty nice hideout. I was tempted to add shutters over the windows but I don't want to go shutter crazy on it.

Cartographer-D commented:
> 
> 
> I'm very not sure about the mulebot staging room and tool storage.
> I suggest you remove the staging room, move the mulebots into cargo (replacing the 2 tables near the crate recycling belt), and expand the tool storage upwards.
> 
> Here's a general idea of what i think tool storage should look like. You can get rid of the middle table and make the room less wide while readding the maintenance path if you want.
> ![image](https://user-images.githubusercontent.com/25937686/79694746-9374a800-8272-11ea-91c7-51cb717b03fe.png)

Having the mulebots right next to a maintenance area is important so they don't go running through the middle of everyone.

The intent is to have the tool storage be compact minus the YouTool. It should discourage metarushing tool storage and cleaning it out OR result in a bit of a struggle/conflict to do so.

Son-of-Space committed:
Removed Hiding Neutral Corners

Got rid of some neutral corners that marked the old layout in the primary hallway.

Fabian272 commented:
Praise this PR, meta cargo was always a bit weird but the tool storage looks good now too. And oh lord if one of those insulated glove pairs arent budget then i will praise this even more

Cartographer-D commented:
> 
> 
> Praise this PR, meta cargo was always a bit weird but the tool storage looks good now too. And oh lord if one of those insulated glove pairs arent budget then i will praise this even more

Real assistants are willing to shock themselves on doors to find the wires.

MarioWizard119 commented:
Could we see pictures of where all of the departmental chutes end? We already have one for Medbay, what about the others?

 I don’t see the departmental cargo chutes being more of a security risk than they already are as long as they only go to the very front of the department. You can already deliver bombs deeper into departments via package wrapping and destination taggers. Though it is interesting to see that the delivery room and the autolathe room are combined into a single room. You’ve basically turned cargo into warp zone. Hop down a trash chute, pick up some things from the lathe, and hop back into the chute for your department. This isn’t necessarily a bad thing, it’d certainly be interesting to see how it’d play out.

Megarop commented:
The ORM vends materials into a wall. You can't pick them up unless you're on cargo's side of the ORM because there's another wall blocking it.

MarioWizard119 commented:
They can still be taken out of a protolathe, but yeah this is a pretty big unga moment.

MrDoomBringer commented:
Yeah. Easiest fix is just changing the input and output dir of the ORM on this map to be EAST/WEST instead of NORTH/SOUTH

MrDoomBringer commented:
You can change the direction of the ORM's input/outputs by altclicking on it with it's panel open, but yeah pretty unga

Cartographer-D commented:
Dropping a fix PR for all the problems I've found within the hour.
